### DISCUSSION TITLE
Add "Users" Field Type for Project Custom Fields

### DISCUSSION THREAD

**Discussion by TylerMatteo** [community/community#122989] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

In addition to the existing custom field types, I would like to add a custom field to my Project that allows me to select from Github users associated with the project.
<img width="369" alt="image" src="https://github.com/community/community/assets/9055367/f327e322-eb61-408d-862f-afdab5bc4bd3">

This would be similar to how "Assignees" are associated to Issues, and could support one or many users. The "assignee" field works for my use cases with some work around but it would be great if we could assign users with custom fields. The main use case I had in mind would be so that we could indicate who is assigned for engineering vs QA vs design, etc.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by iwahbe** [comment]
Looks like this is a duplicate of https://github.com/orgs/community/discussions/10045.

**Comment by TylerMatteo** [comment]
It is 😞 I didn't see that when I was looking. Closing this as dupe.

**Comment by jarpoole** [comment]
@ankneis because the "assignees" and "reviewers" fields are currently part of the issue under the hood, would we see this change implemented as part of https://github.com/github/roadmap/issues/1088?
